pkg/reconciler/revision/table_test.go

```diff
@@ -61,118 +61,6 @@ func TestReconcile(t *testing.T) {
 		Name: "key not found",
 		// Make sure Reconcile handles good keys that don't exist.
 		Key: "foo/not-found",
-	}, {
```

Comment:
Why are all these deleted?

Reply:
Sorry it was my mistake unknowingly those test cases deleted 
reverted back Thank you